### PR TITLE
check os compatibility and dependencies_version in prepare_env.sh

### LINKED ISSUE #93: Improve "Getting Started" experience
1. Improve `./scripts/prepare_env.sh` script to check compatibility.
   - Check version of the OS (for example, WSL1 or WSL2 of Windows)
   - Check versions of dependencies
   ...
   Tell user in a friendly way what is the problem if requirements are not met
2. Improve READEM and Docs to reflect latest compatibility status

### PR DESCRIPTION
**Pull request by acezen**
## What do these changes do?

check os compatibility and dependencies_version in prepare_env.sh to improve "Getting Started" experience

## Related issue number

Fixes #93

### CODE REVIEW

**Comment by acezen** [comment]
cc/  @yecol @sighingnow @wenyuanyu

**Review by sighingnow — APPROVED** [review]
LGTM.

Not part of this task, but I'm wondering if we suggest users kind rather than minikube on Linux and MacOS as well. Test is needed, though. (not in high pri).

File: README.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ Read more in the [whitepaper](https://github.com/alibaba/GraphScope/blob/main/do
...
-To run GraphScope on your local computer with minikube, the following dependencies or tools are required.
+To run GraphScope on your local computer, the following dependencies or tools are required.
 
 - Docker
-- minikube
 - Python >= 3.6 (with pip)
+- Local Kubernetes cluster set-up tool (e.g. [Minikube](https://minikube.sigs.k8s.io) or [Kind](https://kind.sigs.k8s.io))
 
-For Linux distributions, we provide a script to install the above dependencies and prepare the environment.
+For Ubuntu/CentOs/WSL2, we provide a script to install the above dependencies and prepare the environment.
```
Comment (yecol) [inline README.md:31]: CentOS

**Review by yecol — CHANGES_REQUESTED** [review]
You may also want to revise the file in `/docs`, for  https://graphscope.io/docs/installation.html

**Comment by wenyuanyu** [comment]
Maybe also checking the platform's version? like Ubuntu 14, 16, 18?

**Review by yecol — CHANGES_REQUESTED** [review]

**Comment by acezen** [comment]
> LGTM.
> 
> Not part of this task, but I'm wondering if we suggest users kind rather than minikube on Linux and MacOS as well. Test is needed, though. (not in high pri).

thanks for the suggestion. this script can include MacOS prepare environment though kind i think, i will add this feature in next related pr after test.

**Review by yecol — APPROVED** [review]

**Pull request merged**